2 - RTB/Documentazione interna/Verbali/Interni/23-11-13.typ

```diff
@@ -0,0 +1,65 @@
+#import "/template.typ": *
+
+#show: project.with(
+  title: "Verbale 13/11/23",
+  docType: "verbale",        
+  partecipants: (
+    (name: "Banzato Alessio", role: "Verificatore"),
+    (name: "Carraro Riccardo", role: "Verificatore"),
+    (name: "Gardin Giovanni",  role: "Redattore"),
+    (name: "Nardo Silvio",   role: "Redattore"),
+    (name: "Oseliero Antonio",   role: "Amministratore"),
+    (name: "Todesco Mattia",  role: "Verificatore"),
+    (name: "Zaccone Rosario",  role: "Amministratore"),
+  ),
+  oraInizio: "15:30",
+  oraFine:  "17:30",
+)
+
+#set text(font:"New Computer Modern", size: 1.3em)
+#set align(left)
+
+
+
+= Ordine del giorno
+  - Aggiornamento del gruppo in merito alle nuove automazioni adottate;
+  - Retrospettiva dello sprint 2;
+  - Discussione in merito alle tecnologie adottate;
+  - Ripartizione dei ruoli per il nuovo sprint;
+  - Pianificazione sprint 3;
+
+== Automazioni e nuova repository
+  Il gruppo che si è dedicato nel precedente sprint ad automatizzare la repository ha 
+  finalizzato il lavoro e ha aggiornato il gruppo riguardo alle nuove funzionalità.
+  La vecchia repository è stata abbandonata in favore di una nuova (questo perchè riorganizzare 
+  la vecchia repository avrebbe richiesto troppo tempo) con una nuova organizzazione dei branch, 
+  differente rispetto alla precedente. La differenza fondamentale è prima si aveva 
+  un branch main per le release, un branch develop comune da cui si aprivano poi i vari branch feature 
+  di lavoro; ora la repository è composta da tre branch principali quali:
+  - main: contenente i pdf compilati e pubblicati automaticamente tramite github actions solo a seguito della verifica di uno o più reviwer;
+  - src: contenente i file sorgenti scritti in Typst;
+  - website: branch parallello contente un versione work in progress del sito del team.
+
+
+== Retrospettiva sullo sprint 
+  Segue quindi la discussione in merito ai miglioramenti da attuare nei prossimi sprint e quello 
+  attuale. Lo sprint passato ha avuto due criticità fondamentali: una generale confusione da 
+  parte del team di documentazione in merito alla priorità da dare ai vari documenti da redigere 
+  e al loro contenuto; l'impossibilità di utilizzare la nuova repository bloccata dai 
+  rallentamenti subiti dal sottovalutare la mole di lavoro riguardo l'implementazione delle automazioni.
+
```

Comment:
Aggiungerei cosa abbiamo fatto per risolvere queste due criticità:
- stabilito le priorità dei documenti e divisione del team in sottogruppi
- terminata l’implementazione delle github action 
(Ovviamente rendendo questi due punti un po’ più discorsivi)